### PR TITLE
Freemabd1/rw 351 implemented api call to return gender, race, ethnicity lists.

### PR DESCRIPTION
**Pull request by freemabd1**
implemented api call to return gender, race, ethnicity lists.

### CODE REVIEW

File: api/src/main/resources/workbench.yaml [hunk 1]
```diff
@@ -935,6 +935,23 @@ paths:
...
+      - $ref: '#/parameters/cohortId'
+      - $ref: '#/parameters/cdrVersionId'
+    get:
+      tags:
+        - cohortReview
+      description: Will return a list all values for gender, race and ethnicity.
+      operationId: "getParticipantFilters"
+      responses:
+        200:
+          description: A cohortReviewId and cohort count
```
Comment (johnstill) [inline api/src/main/resources/workbench.yaml]: This description is outdated (probably copy/pasted?)

File: api/src/main/resources/workbench.yaml [hunk 2]
```diff
@@ -935,6 +935,23 @@ paths:
...
+    parameters:
+      - $ref: '#/parameters/workspaceNamespace'
+      - $ref: '#/parameters/workspaceId'
+      - $ref: '#/parameters/cohortId'
+      - $ref: '#/parameters/cdrVersionId'
+    get:
+      tags:
+        - cohortReview
+      description: Will return a list all values for gender, race and ethnicity.
+      operationId: "getParticipantFilters"
```
Comment (johnstill) [inline api/src/main/resources/workbench.yaml]: I'm not super in love with calling these "Filters" - that's what they'll be _used in_, not what they _are_, but a better name doesn't occur to me atm.  I'd say this is good enough for now and probably low hanging fruit for a rename / refactor down the road.

File: api/src/main/resources/workbench.yaml (hunk 1 above)
Comment (freemabd1) [reply api/src/main/resources/workbench.yaml]: yep.. done.

File: api/src/main/resources/workbench.yaml (hunk 2 above)
Comment (freemabd1) [reply api/src/main/resources/workbench.yaml]: getParticipantDemographics?

**Pull request merged**